tutorials/Tutorial15_TableQA.ipynb

```diff
@@ -88,7 +88,7 @@
   },
```

Comment:
<p>This output should be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/2325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>